boards/cats_rev1Pro/Core/Src/tasks/task_init.c

```diff
@@ -339,6 +343,13 @@ static void init_baro() {
   osDelay(10);
 }
 
+static void init_magneto() {
+  spi_init(MAG.spi);
+  mmc5983ma_selftest(&MAG);
+  mmc5983ma_init(&MAG);
+  // mmc5983_calibration(&MAG);
```

Comment:
The calibration is very important to get a well working orientation

Reply:
Will open an issue for that